.github/workflows/ci.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.2, 7.3]
+                php: [7.3, 7.4]
```

Comment:
Aren’t the test runs for 7.1 and 7.2 missing?

Reply:
We never had `7.1`. And I had to ditch `7.2` in order to be compatible with PhpUnit 9.3. 🤷🏻

Reply:
I don’t like having supported PHP versions that we don’t run the test suite on. I constantly forget which features are available since which PHP version and the test suite always reminds me of it ☺️

In `contao/image` we are using `"phpunit/phpunit": "^7.5 || ^8.0"` to be compatible with more php versions. Did you try if this is possible with this codebase too?

Reply:
I will try. 👍🏻

Reply:
If I understood correctly Symfony uses the phpunit-bridge for exactly this problem (including forward compatible polyfills). Maybe we could use this as well.

Reply:
Re-added in 7a7c53ed075ce196683674dffc8e8b7bb4abc966.

Reply:
Can we add 7.1 too? 🥺

Reply:
No. PHP 7.1 has already reached its EOL and we have not tested it before, either.

Reply:
And the [InvokedCount](https://github.com/sebastianbergmann/phpunit/blob/8.5/src/Framework/MockObject/Rule/InvokedCount.php) class has only been added in PhpUnit 8.5, so to test PHP 7.1, we would have to lower the PhpUnit version constraint and add `class_exists()` if conditions to the tests – only to test an outdated PHP version?